### DISCUSSION TITLE
The Week 7... (Huge Update)

### DISCUSSION THREAD

**Discussion by ghost** [luckydog7/Funkin-android#1463] [mods]
FINALLY (But Beta) 
by gamerbross

link:https://cntr.click/V2SZG4z

Things Coming:
Tankman Ugh And Pretty Good Animation
Smoke Effects
Stress Tankmans and Death Animations

Unfortunately, Stress Won't Be Pico

Removed Things:
Cutscenes (Reason: 200mb)
No Gf Replaced GfTankman

enjoy! (I am Turkish)

**Comment by Kappandrew** [comment]
Mfw im one of the 15 or so ppl with a decent phone in this community

**Comment by ghost** [comment]
> Mfw im one of the 15 or so ppl with a decent phone in this community

nice
